frontend/app/craco.config.js

```diff
@@ -82,23 +91,27 @@ module.exports = {
         // turn off sourcemaps
         webpackConfig.devtool = "eval"
       } else {
-        minimizerPlugins[terserPluginIndex].options.parallel = true
+        // TODO: Trace/Test this for update to GITHUB_ACTION instead
+        const parallel = process.env.CIRCLECI ? false : true
+        minimizerPlugins[terserPluginIndex].options.parallel = parallel
       }
 
-      // detect circular dependency plugins as these can cause runtime errors
-      // no circular dependencies also mean better jest test runtimes, code organization, and better code splitting
-      webpackConfig.plugins.push(
-        new CircularDependencyPlugin({
-          // exclude detection of files based on a RegExp
-          exclude: /node_modules/,
-          // add errors to webpack compiltion instead of warnings
-          failOnError: true,
-          // allow import cycles that include an asyncronous import,
-          // e.g. via import(/* webpackMode: "weak" */ './file.js')
-          allowAsyncCycles: false,
-        })
+      // Remove ModuleScopePlugin which throws when we try to import something
+      // outside of src/.
+      webpackConfig.resolve.plugins.pop()
```

Comment:
I would prefer a more specific search to remove by name. See further up in the code.

This will make this more resilient to react-scripts.